packages/email/email.js

```diff
@@ -134,7 +134,7 @@ EmailTest.hookSend = function (f) {
  * @param {String} [options.text|html] Mail body (in plain text and/or HTML)
  * @param {String} [options.watchHtml] Mail body in HTML specific for Apple Watch
  * @param {String} [options.icalEvent] iCalendar event attachment
- * @param {Object} [options.headers] Dictionary of custom headers
+ * @param {Object} [options.headers] Dictionary of custom headers - e.g. `{ headerName: 'header value' }`. To set an object under a header name, use `JSON.stringify` - e.g. `{ headerName: JSON.stringify({ tracking: { level: 'full' } }) }`.
```

Comment:
Should we use `"header name": "header value"` here to emphasize that the header name is variable, and `headerName` isn't a special field? (Is that even a good way to emphasize that?)

Reply:
Good point - done - thanks!